code/analyze/ac/stats/de_StatsFreqPreferences.m

```diff
@@ -79,9 +84,61 @@
         stats.(prop).std{si}(mi,:,:) = std(resps_by_prop, [], 3);
       end;
 
-      % Now, do the analysis as done in the `recfields` scripts.
-
-
     end;
-  end;
+    % Now, do the analysis as done in the `recfields` scripts.
+    
+    %Take raw stds and average over all units (see nn_2layer_processor)
+    raw_std = stats.freq.std{si};
+    %First, combine first 2 dimensions (models x hidden units)
+    raw_std = reshape(raw_std, [size(raw_std, 1)*size(raw_std, 2), size(raw_std, 3)]);
+    sm = mean(raw_std,1); %avg over each unit, like in recfields
+    ss  = std(raw_std,[],1)/sqrt(size(raw_std,1)); %stick to naming convention
```

Comment:
This is actually stderr. Can you updates names here, and in the other script, to reflect that?